ocs_ci/ocs/replica_one.py

```diff
@@ -0,0 +1,199 @@
+from logging import getLogger
+
+from ocs_ci.framework import config
+from ocs_ci.ocs.resources.pod import (
+    get_pods_having_label,
+    get_ceph_tools_pod,
+    run_osd_removal_job,
+)
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.constants import (
+    DEFAULT_CEPHBLOCKPOOL,
+    OSD_APP_LABEL,
+    CEPHBLOCKPOOL,
+    STORAGECLASS,
+    DEPLOYMENT,
+)
+from ocs_ci.ocs.exceptions import CommandFailed
+
+
+log = getLogger(__name__)
+
+REPLICA1_STORAGECLASS = "ocs-storagecluster-ceph-non-resilient-rbd"
+
+# For local testing only. to be removed
+config.ENV_DATA["worker_availability_zones"] = [
```

Comment:
Please implement reading this from config file.

Reply:
it is. see comment above.

Reply:
As there are missing implementation for some functions please add WIP again till the PR will be ready to run.